src/rippled/lib/streams.js

```diff
@@ -44,6 +44,16 @@ const fetchLedger = (ledger, attempts = 0) => {
     });
 };
 
+const fetchServerInfo = () => {
```

Comment:
Since this just returns the fee, maybe it should have a different name like `fetchServerFee`?